src/main/resources/assets/minecolonies/lang/en_US.lang

```diff
@@ -77,105 +69,77 @@ tile.minecolonies.blockHutMiner.name=Mine
 tile.minecolonies.blockHutStonemason.name=Quarry
 tile.minecolonies.blockHutTownHall.name=Town Hall
 tile.minecolonies.blockSubstitution.name=SchematicPlaceholder.
-
 tile.blockHut.messageNoPermission=You don't have permission to open huts in %s.
 tile.blockHut.messageNoPermissionPlace=You don't have permission to place huts in %s.
-
 com.minecolonies.job.Builder=Builder
 com.minecolonies.job.Deliveryman=Deliveryman
 com.minecolonies.job.Farmer=Farmer
 com.minecolonies.job.Lumberjack=Lumberjack
 com.minecolonies.job.Miner=Miner
 com.minecolonies.job.Fisherman=Fisherman
-
 tile.blockHutTownHall.messageMaxSize=Colony has reached max size
 tile.blockHutTownHall.messageColonistDead=A colonist has died!
 tile.blockHutTownHall.messagePlacedAlready=You have placed a Town Hall already
 tile.blockHutTownHall.messageInvalidWorld=You can't place a town hall in this world type
 tile.blockHutTownHall.messageTooClose=Too close to existing town hall
 tile.blockHutTownHall.messageTooFar=Too far away from your Colony
-
 tile.blockHut.messageTooFarFromTownHall=You need to be closer to your town hall!
 tile.blockHut.messageNoTownHall=You need to place a town hall first!
-
 item.minecolonies.supplyChestDeployer.name=Supply Chest
 item.minecolonies.scepterGold.name=Building Tool
 item.minecolonies.scepterSteel.name=Scan Tool
 item.minecolonies.caliper.name=Caliper
-
 item.supplyChestDeployer.invalid=You must be near a big pool of water
-
 item.scepterSteel.point=Point one saved
 item.scepterSteel.point2=Point two saved, right click again to save the schematic
 item.scepterSteel.samePoint=That was the same point
 item.scepterSteel.scanFormat=SCAN_%s-%s.schematic
 item.scepterSteel.scanSuccess=Scan successfully saved as %s
 item.scepterSteel.scanFailure=Scan failed to save
-
 item.caliper.message.same=That's the same block.
 item.caliper.message.line=That's a %d block line.
 item.caliper.message.square=That's a %d by %d square.
 item.caliper.message.cube=That's a %d by %d by %d cube.
-
 itemGroup.minecolonies=MineColonies
-
 entity.worker.inventoryFullChestFull=Please make space in my building chest. I am not able to dump my inventory.
-
 entity.builder.messageBuildStart=Started building %s
 entity.builder.messageBuildComplete=Finished building %s
 entity.builder.messageNeedMaterial=I need %s (Amount: %d)
 entity.builder.messageBuilderNecessary=Requiring a builder of level %d in order to build or upgrade this building.
-entity.builder.messageNoBuilder=You need a builder in order to build or upgrade this building.
```

Comment:
Seems so. We cannot import and export at the same time. @Raycoms has to add the new strings first (or tell me from which language to import them ;) )